modules/behaviors/DataBehavior.js

```diff
@@ -774,6 +820,36 @@
     },
 
     /**
+     * Determine if behavior is loading ids or objects.
+     * @method isLoading
+     * @return {Boolean} true - the behavior is loading objects or ids.
+     *                   false - the behavior is not loading objects or ids.
+     */
+    isLoading: function() {
```

Comment:
just wondering how there's two isLoading methods in this file. It looks like they're different object, maybe a view and behavior? idk.

Reply:
One is the Data object the other is the Behavior.  new (DataBehavior()).data is a full-fledged private object defined in DataBehavior.